airbyte-integrations/connectors/destination-exasol/src/main/java/io/airbyte/integrations/destination/exasol/ExasolSqlOperations.java

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright (c) 2022 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.integrations.destination.exasol;
+
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+
+import io.airbyte.db.jdbc.JdbcDatabase;
+import io.airbyte.integrations.base.JavaBaseConstants;
+import io.airbyte.integrations.destination.jdbc.JdbcSqlOperations;
+import io.airbyte.protocol.models.v0.AirbyteRecordMessage;
+
+public class ExasolSqlOperations extends JdbcSqlOperations {
+
+  public static final String COLUMN_NAME_AB_ID =
+      "\"" + JavaBaseConstants.COLUMN_NAME_AB_ID.toUpperCase() + "\"";
+  public static final String COLUMN_NAME_DATA =
+      "\"" + JavaBaseConstants.COLUMN_NAME_DATA.toUpperCase() + "\"";
+  public static final String COLUMN_NAME_EMITTED_AT =
+      "\"" + JavaBaseConstants.COLUMN_NAME_EMITTED_AT.toUpperCase() + "\"";
+
+  @Override
+  public String createTableQuery(final JdbcDatabase database, final String schemaName, final String tableName) {
+    String query = String.format(
+        "CREATE TABLE IF NOT EXISTS %s.%s ( \n"
+            + "%s VARCHAR(64),\n"
+            + "%s VARCHAR(2000000),\n"
+            + "%s TIMESTAMP DEFAULT CURRENT_TIMESTAMP,\n"
+            + "PRIMARY KEY(%s)\n"
+            + ")\n",
+        schemaName, tableName,
+        ExasolSqlOperations.COLUMN_NAME_AB_ID,
+        ExasolSqlOperations.COLUMN_NAME_DATA,
+        ExasolSqlOperations.COLUMN_NAME_EMITTED_AT,
+        ExasolSqlOperations.COLUMN_NAME_AB_ID);
+    LOGGER.info("Create table query: {}", query);
+    return query;
+  }
+
+  @Override
+  public void executeTransaction(final JdbcDatabase database, final List<String> queries) throws Exception {
```

Comment:
the `executeTransaction` method is supposed to execute the queries in a transactional way. either all of them succeeded or the operation is rolled back. Transactions seem possible with exasol https://docs.exasol.com/db/latest/database_concepts/transaction_management.htm

Reply:
In order to use transactions I need to deactivate autocommit by using `autocommit=0` in `getDefaultConnectionProperties()`. This will affect all Exasol connections in Airbyte. Will this be a problem?

Reply:
hmm would you mind explaining why would it be a problem?

Reply:
I don't know how Airbyte works internally and if it assumes autocommit to be enabled (which is the default for Exasol). But if you say it's OK then it's not problem :-)

Reply:
Hmm I don't think that Airbyte assumes anything, it should be up to the connector. If the change doesn't affect the rest of the connectors and it is only relevant for Exasol then I don't see any issues.